tests/python/relay/test_pass_merge_composite.py

```diff
@@ -64,55 +65,53 @@
 
 
 def make_add_sub_mul_pattern():
-    """Create a pattern to match the following graph.
+    r"""Create a pattern to match the following graph.
 
         add  sub
          \   /
           \ /
           mul
     """
-    x = relay.var('x')
-    y = relay.var('y')
-    add_node = relay.add(x, y)
-    sub_node = relay.subtract(x, y)
-    mul_node = relay.multiply(add_node, sub_node)
+    x = wildcard()
+    y = wildcard()
+    add_node = is_op('add')(x, y)
+    sub_node = is_op('subtract')(x, y)
+    mul_node = is_op('multiply')(add_node, sub_node)
```

Comment:
Just FYI, if you think it would be easier, the DFPattern objects on the python side have syntatic sugar for +, -, *, / , |

Reply:
Wow that's nice to know. I'll change some patterns using this syntax to demonstrate the different ways of making patterns.